test/audiovideofacade/DefaultAudioVideoFacade.test.ts

```diff
@@ -368,8 +368,14 @@ describe('DefaultAudioVideoFacade', () => {
         'realtimeUnsubscribeFromVolumeIndicator'
       );
       const arg1 = '';
-      facade.realtimeUnsubscribeFromVolumeIndicator(arg1);
-      assert(spy.calledOnceWith(arg1));
+      const arg2 = (
+        _attendeeId: string,
+        _volume: number | null,
+        _muted: boolean | null,
+        _signalStrength: number | null
+      ): void => {};
+      facade.realtimeUnsubscribeFromVolumeIndicator(arg1, arg2);
```

Comment:
Should we create another test case for the optional parameter instead of replace the existing one?

Reply:
Yep, I agree. Updating.